desc/compute/_field.py

```diff
@@ -2220,12 +2238,13 @@ def _B_mag(params, transforms, profiles, data, **kwargs):
     ],
 )
 def _B_mag_r(params, transforms, profiles, data, **kwargs):
-    data["|B|_r"] = (
+    data["|B|_r"] = safediv(
```

Comment:
were there cases where |B| was zero?

Reply:
Yes, after coordinate mappings
```python
eq = get("W7-X")
rho = np.linspace(0, 1, 5)
alpha = np.linspace(0, 2 * np.pi, 2)
zeta = np.linspace(0, 4 * np.pi, 10)
grid, data = field_line_to_desc_coords(eq, rho, alpha, zeta)
data = eq.compute(["|B|", "|B|_z constant rho alpha"], grid=grid, data=data)
print(data["|B|"])
```

Reply:
I think that might be indicative of some sort of bug. I don't think `|B|` should be zero anywhere in the plasma volume, regardless of what coordinates are used.